roles/confluent.ksql/tasks/rbac.yml

```diff
@@ -85,6 +85,11 @@
             "resourceType":"Group",
             "name":"_confluent-ksql-{{ksql_final_properties['ksql.service.id']}}",
             "patternType":"PREFIXED"
+          },
+          {
+            "resourceType":"Topic",
+            "name": "_confluent-ksql-{{ksql_final_properties['ksql.service.id']}}",
+            "patternType": "LITERAL"
```

Comment:
This should be PREFIXED - apologies, this one is on me, my jira was not clear enough

Reply:
Updated.